app/test_main.py

```diff
@@ -1,3 +1,66 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "input_value_cat,input_value_dog,expected_value",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="does not work correctly with the value zero"
+            ),
+            pytest.param(
+                100,
+                100,
+                [21, 17],
+                id="does not work correctly with a large value"
+            ),
+            pytest.param(
+                23,
+                23,
+                [1, 1],
+                id="does not work correctly with a normal value"
+            ),
+            pytest.param(
+                -2,
+                -3,
+                [0, 0],
+                id="does not work correctly with a negative value"
+            ),
+            pytest.param(
+                27,
+                27,
+                [2, 2],
+                id="does not work correctly with a normal value"
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="does not work correctly with a normal value"
+            ),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="does not work correctly with a normal value"
+            ),
+            pytest.param(
+                28,
+                28,
+                [3, 2],
+                id="does not work correctly with a normal value"
```

Comment:
Why id="does not work correctly with a normal value" in good cases?
It works correctly